.travis.yml

```diff
@@ -1,17 +1,45 @@
-language: csharp
-mono: none
+language: php
```

Comment:
I don't think this is the way to do it. Since Travis is language specific, we need to change the architecture or our Github repos.

Reply:
The current architecture only works for 1 language at a time. This was changed in order to make the PHP work. Reverting this change will stop PHP CI from working.